Documentation/network/concepts/ipam/multi-pool.rst

```diff
@@ -70,7 +70,10 @@ PodCIDRs are allocated:
 New pools can be added at run-time. The list of CIDRs in each pool can also be
 extended at run-time. In-use CIDRs may not be removed from an existing pool, and
 existing pools may not be deleted if they are still in use by a Cilium node.
-The mask size of a pool is immutable and the same for all nodes.
+The mask size of a pool is immutable and the same for all nodes. Neither restriction
+is enforced until `Issue #26966`_ is resolved.
+
+.. _Issue #26966: https://github.com/cilium/cilium/issues/26966
```

Comment:
We have a sphinx extension for this:

```suggestion
is enforced until :gh-issue:`26966` is resolved.
```

Reply:
Thanks for the heads-up. Commit 9548223 resolves this comment.